src/libpmem/pmem.c

```diff
@@ -661,10 +860,10 @@ pmem_memmove(void *pmemdest, const void *src, size_t len)
 }
 
 /*
- * pmem_memcpy -- memcpy to pmem
+ * pmem_memcpy_persist -- memcpy to pmem
  */
 void *
-pmem_memcpy(void *pmemdest, const void *src, size_t len)
+pmem_memcpy_persist(void *pmemdest, const void *src, size_t len)
```

Comment:
I can't seem to leave a comment on line 159, but there are three instances of the old names in a comment that need updating.